src/test/resources/testfiles/testfile2

```diff
@@ -0,0 +1 @@
+h
```

Comment:
Minor thing, but it's easier to understand what the purpose of this file is if you make the content something like "Placeholder file content" or "Example test file". When it's just "h", my reaction was to search for this file to make sure it was actually intentional rather than something that was committed by accident. 🙂

Reply:
Yeah good point, I've changed it to Placeholder file content.